etc/profile-m-z/mcomix.profile

```diff
@@ -0,0 +1,71 @@
+# Firejail profile for mcomix
+# Description: A comic book and manga viewer 
+# This file is overwritten after every install/update
+# Persistent local customizations
+include mcomix.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/mcomix
+noblacklist ${HOME}/.local/share/mcomix
+noblacklist ${DOCUMENTS}
+
+# Allow /bin/sh (blacklisted by disable-shell.inc)
+include allow-bin-sh.inc
+
+# Allow python (blacklisted by disable-interpreters.inc)
+#1.2
```

Comment:
1.2?

Reply:
version 1.2 is the last version with python 2 and gtk2